src/components/ContainerHeader.vue

```diff
@@ -396,7 +417,7 @@ export default {
 
 .kiwi-header-notjoined .u-link {
     font-weight: 600;
-    line-height: 45px;
+    line-height: 43px;
```

Comment:
Is this 43px because its the height -1px? If so it could do with a comment explaining that